apps/bfd-pipeline/bfd-pipeline-rif-load/src/test/java/gov/cms/bfd/pipeline/rif/load/RifLoaderIT.java

```diff
@@ -557,10 +557,9 @@ private void loadSample(DataSource dataSource, List<StaticRifResource> sampleRes
 
     // Verify that the expected number of records were run successfully.
     Assert.assertEquals(0, failureCount.get());
-    Assert.assertEquals(
-        "Unexpected number of loaded records.",
-        sampleResources.stream().mapToInt(r -> r.getRecordCount()).sum(),
-        loadCount.get());
+    Assert.assertTrue(
+        "The number of loaded records should meet or exceed the number of expected records. The expected records can be exceeded because of line items.",
+        (loadCount.get() >= sampleResources.stream().mapToInt(r -> r.getRecordCount()).sum()));
```

Comment:
Note to self: need to figure out why this change was needed; it shouldn't be (if I'm reading things correctly).